python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -1425,59 +1595,64 @@ def run(self, ccdExposure, *, dnlLUT=None, bias=None, deferredChargeCalib=None,
                 ccdExposure,
             )
 
+        # Linearity correction
+        # Units: electrons
+        if self.config.doLinearize:
+            self.log.info("Applying linearizer.")
+            # The linearizer is in units of ADU.
```

Comment:
Can't the new linearizer have whichever units, so shouldn't this check to see if there's a mismatch based on the linearizer header value?

Reply:
Because of the order in which we construct calibrations, it would be hard/impossible/circular to have an electron linearizer.  We should discuss.